Backpack/Alert/Classes/BPKAlertController.m

```diff
@@ -0,0 +1,167 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKAlertController.h"
+#import <Backpack/Color.h>
+#import <Backpack/Spacing.h>
+#import <Backpack/Font.h>
+#import "BPKAlertView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BPKAlertController()
+
+@property (nonatomic, strong) UIView *faderView;
+@property (nonatomic, strong) UIButton *doneButton;
+@property (nonatomic, strong) BPKAlertView *alertView;
+@property (nonatomic, strong) UIWindow *window;
+
+@end
+
+@implementation BPKAlertController
+{
+    BPKAlertConfiguration *_configuration;
+}
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        _window = [[UIApplication sharedApplication] keyWindow];
+        
+        _faderView = [UIView new];
```

Comment:
Can we prefer property usage over direct ivars outside of ovewritten properties?